ambry-api/src/main/java/com/github/ambry/account/Container.java

```diff
@@ -478,6 +498,7 @@ JSONObject toJson() throws JSONException {
         metadata.put(JSON_VERSION_KEY, JSON_VERSION_1);
         metadata.put(CONTAINER_ID_KEY, id);
         metadata.put(CONTAINER_NAME_KEY, name);
+        metadata.put(CONTAINER_LAST_UPDATE_TIME_KEY, lastUpdateTime);
```

Comment:
please don't change the current version.

Reply:
Removed put for version one.